### ISSUE TITLE
Two presses of the Tab key are required to move keyboard focus from the Open Navigation button to the first focusable control that follows it

### ISSUE THREAD

**Issue by gbarkerz** [dotnet/maui#8963]
### Description

Repro app is https://github.com/gbarkerz/MyVeryFirstMauiApp.

In the repro app, from a user's perspective, the first focusable control that follows the Open Navigation hamburger button in the Tab order is a button called "Test". As such, a single press of the Tab key should move keyboard focus from the Open Navigation button to the Test button. Instead the first press of the Tab key seems to have no effect, and then the second Tab key press moves keyboard focus. 

### Steps to Reproduce

1. Build and run the repro app for Windows.
2. Tab to the Open Navigation button.
3. Press Tab.

Result->Nothing seems to happen.
Expected->Keyboard focus moves to the Test button.

.NET version is 6.0.400-preview.22330.6.

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

net6.0-windows10.0.19041.0

### Did you find any workaround?

Press the Tab key twice instead of once.

**Comment by PureWeen** [comment]
This might be resolved by the following issue

https://github.com/dotnet/maui/pull/9270

Once we have NET7 bits out into the public you should be able to test

**Comment by fsteffi** [comment]
GitHubTags:
#.NET MAUI; #A11ySev1; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG2.1.1; #GH_.NETMAUI_Win32_Apr2022; #AILimited​; #Win32; #Keyboard;
